src/sugar3/graphics/palette.py

```diff
@@ -223,7 +223,19 @@ def popup(self, immediate=False):
 
         PaletteWindow.popup(self, immediate)
 
-    def popdown(self, immediate=False):
+    def popdown(self, immediate=False, state=None):
+        '''
+        Popdown (or show the full contents of) the palette.
+
+        Keyword Args:
+            immediate (bool): if True, the palette will be shown instantly (as
+                if the user right clicked the item).  If False, the palette
+                will be shown after the usual activation wait time.
+
+        .. deprecated:: 0.109.3
```

Comment:
There's no 0.109.3 in your [release roadmap](https://wiki.sugarlabs.org/go/0.110/Roadmap), perhaps you mean 0.109.2.0 feature freeze on 15th, or just use 0.109?